cmd/runmqserver/qmgr.go

```diff
@@ -41,6 +41,38 @@ func createDirStructure() error {
 	return nil
 }
 
+// configureOwnership recursively handles ownership of files within the given filepath
+func configureOwnership(paths []string) error {
+	uid, gid, err := command.LookupMQM()
+	if err != nil {
+		return err
+	}
+	for _, root := range paths {
+		_, err = os.Stat(root)
+		if err != nil {
+			if os.IsNotExist(err) {
+				return nil
+			}
+			return err
+		}
+		err = filepath.Walk(root, func(from string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			to := fmt.Sprintf("%v%v", root, from[len(root):])
+			err = os.Chown(to, uid, gid)
```

Comment:
I wonder if it would be safer to only run "chown" if necessary.  How confident are you that the files should always be `mqm:mqm`?

Reply:
All files on a new qm we're owned by mqm but added uid == 999 check to be safe